src/table_translation.json

```diff
@@ -0,0 +1 @@
+{"Tx":{"0":0,"1":10,"2":0,"3":-10,"4":0,"5":0,"6":0,"7":0,"8":0,"9":0,"10":0,"11":0,"12":0},"Ty":{"0":0,"1":0,"2":0,"3":0,"4":0,"5":10,"6":0,"7":-10,"8":0,"9":0,"10":0,"11":0,"12":0},"Tz":{"0":0,"1":0,"2":0,"3":0,"4":0,"5":0,"6":0,"7":0,"8":0,"9":10,"10":0,"11":-10,"12":0},"At1":{"0":0,"1":0,"2":0,"3":0,"4":0,"5":0,"6":0,"7":0,"8":0,"9":0,"10":0,"11":0,"12":0},"At2":{"0":0,"1":0,"2":0,"3":0,"4":0,"5":0,"6":0,"7":0,"8":0,"9":0,"10":0,"11":0,"12":0},"At3":{"0":0,"1":0,"2":0,"3":0,"4":0,"5":0,"6":0,"7":0,"8":0,"9":0,"10":0,"11":0,"12":0},"FS_lat":{"0":20,"1":20,"2":20,"3":20,"4":20,"5":20,"6":20,"7":20,"8":20,"9":20,"10":20,"11":20,"12":20},"FS_long":{"0":20,"1":20,"2":20,"3":20,"4":20,"5":20,"6":20,"7":20,"8":20,"9":20,"10":20,"11":20,"12":20},"DSD":{"0":107,"1":107,"2":107,"3":107,"4":107,"5":107,"6":107,"7":107,"8":107,"9":107,"10":107,"11":107,"12":107},"DSI":{"0":78.5,"1":78.5,"2":78.5,"3":78.5,"4":78.5,"5":78.5,"6":78.5,"7":78.5,"8":78.5,"9":78.5,"10":78.5,"11":78.5,"12":78.5},"DID":{"0":28.5,"1":28.5,"2":28.5,"3":28.5,"4":28.5,"5":28.5,"6":28.5,"7":28.5,"8":28.5,"9":28.5,"10":28.5,"11":28.5,"12":28.5},"DSIRP":{"0":63.5,"1":63.5,"2":63.5,"3":63.5,"4":63.5,"5":63.5,"6":63.5,"7":63.5,"8":63.5,"9":63.5,"10":63.5,"11":63.5,"12":63.5},"K_IRP":{"0":0,"1":0,"2":0,"3":0,"4":0,"5":0,"6":0,"7":0,"8":0,"9":0,"10":0,"11":0,"12":0},"DSL":{"0":40,"1":40,"2":40,"3":40,"4":40,"5":40,"6":40,"7":40,"8":40,"9":40,"10":40,"11":40,"12":40},"Ap1":{"0":0,"1":0,"2":0,"3":0,"4":0,"5":0,"6":0,"7":0,"8":0,"9":0,"10":0,"11":0,"12":0},"Ap2":{"0":0,"1":0,"2":0,"3":0,"4":0,"5":0,"6":0,"7":0,"8":0,"9":0,"10":0,"11":0,"12":0},"Ap3":{"0":0,"1":0,"2":0,"3":0,"4":0,"5":0,"6":0,"7":0,"8":0,"9":0,"10":0,"11":0,"12":0}}
```

Comment:
Should we make this human-readable/prettify it?

Reply:
resolved